inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/SimpleGroupStatus.java

```diff
@@ -36,7 +36,6 @@ public enum SimpleGroupStatus {
     public static SimpleGroupStatus parseStatusByCode(int code) {
         GroupStatus groupStatus = GroupStatus.forCode(code);
         switch (groupStatus) {
-            case DRAFT:
```

Comment:
why to delete it?

Reply:
Because this status is not used anywhere.